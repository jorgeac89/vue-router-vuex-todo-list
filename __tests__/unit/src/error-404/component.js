import { shallowMount } from '@vue/test-utils';
import Error404 from 'error-404';

describe('Error404 component', () => {
  it('renders correctly', () => {
    expect(shallowMount(Error404)).toMatchSnapshot()
  });
});
