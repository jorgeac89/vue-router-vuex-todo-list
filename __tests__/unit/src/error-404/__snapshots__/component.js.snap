// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 component renders correctly 1`] = `
<div class="d-flex flex-column justify-content-center text-center text-dark h-100">
  <h2 class="display-2">
    404
  </h2>
  <div class="display-3"><i class="fas fa-dragon mx-3"></i> <i class="fas fa-cat mx-3"></i> <i class="fas fa-pastafarianism mx-3"></i></div>
  <h3 class="font-weight-light mt-3">
    We have found these mythical creatures, but not the page you are looking for.
  </h3>
</div>
`;
